ext/SparseConnectivityTracerDataInterpolationsExt.jl

```diff
@@ -25,42 +26,61 @@ import DataInterpolations:
 # Utilities #
 #===========#
 
+# Get output dimension for parameterizing AbstractInterpolations
+# This function was removed together with the output type parameter
+# in DataInterpolations v8: https://github.com/SciML/DataInterpolations.jl/pull/396
+# TODO use DataInterpolations.output_dim instead, which always returns an integer
+# which is 0 for scalar output.
+function SCT.get_output_dim(u::AbstractVector{<:Number})
```

Comment:
Is there a reason why these are name-spaced under SparseConnectivityTracer?

Reply:
To be able to use them in the tests as well. Which is quite silly. I'll try to get this in DataInterpolations instead, as `output_size` next to `output_dim`.